changelog.json

```diff
@@ -1,5 +1,20 @@
 {
   "releases": {
+    "1.6.3": [
+      "[Added] Provide option to discard when files would be overwritten by a checkout - #6755. Thanks @mathieudutour!",
+      "[Fixed] Context menu doesn't open when right clicking on the edges of files in Changes list - #6296. Thanks @JQuinnie!",
+      "[Fixed] Display question mark image when there is no commit selected in dark theme - #6915. Thanks @say25!",
+      "[Fixed] No left padding for :emoji:/@user/#issue autocomplete forms. - #6895. Thanks @murrelljenna!",
+      "[Fixed] Reinstate missing image and fix illustration to work in the dark theme when there are no local changes - #6894",
```

Comment:
```suggestion
      "[Fixed] Missing image, illustration updated for dark theme when no local changes - #6894",
```

Reply:
@iAmWillShepherd thoughts on this one?

Reply:
I think we can be more concise, but this suggestion isn't super readable to me, so I'm going to make another commit to update it.